src/backend/web/routes/index.js

```diff
@@ -2,12 +2,14 @@ const express = require('express');
 const path = require('path');
 const admin = require('./admin');
 const opml = require('./opml');
+const sse = require('./server-sent-events');
 
 const router = express.Router();
 
 router.use(express.static(path.join(__dirname, '../../../frontend')));
 
 router.use('/admin', admin);
 router.use('/opml', opml);
+router.use('/stream', sse);
```

Comment:
Instead of `stream`, which is pretty generic, what about `update-events` or `feed-updates`?